tools/decompress_baserom.py

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: © 2024 ZeldaRET
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations
+
+import hashlib
+import io
+import struct
+from pathlib import Path
+import argparse
+
+import crunch64
+import ipl3checksum
+import zlib
+
+def decompressZlib(data: bytes) -> bytes:
```

Comment:
Are any OOT roms zlib-compressed?

Reply:
reading some discord history, ique is

Reply:
Indeed, most iQue games use DEFLATE instead of yaz0.
We still don't have a way to match the specific version of DEFLATE used on iQue games yet, but iQue is usually low on everybody's priority list

Reply:
got it, thanks